sample/AirshipSample/android/build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        minSdkVersion = 16
-        compileSdkVersion = 27
-        targetSdkVersion = 26
-        supportLibVersion = "27.1.1"
+        minSdkVersion = 21
+        compileSdkVersion = 28
+        targetSdkVersion = 27
```

Comment:
targetSdkVersion still set to 27 due to https://github.com/facebook/react-native/issues/22024